example_buffer.py

```diff
@@ -177,6 +177,9 @@ def _determine_chunk_to_make(write_dir):
 
     return chunk_to_make, False
 
+def get_window_size(chunk_num):
+    return min(500000, (chunk_num + 5) * MINIMUM_NEW_GAMES)
```

Comment:
Consider documenting where this idea came from.

Reply:
Done.